index.html

```diff
@@ -400,6 +400,12 @@
           entryType names</a> within the `entryTypes` sequence, and replace
           the `entryTypes` sequence with the new filtered sequence.
         </li>
+        <p class="note">In some cases the developer may want to distinguish
```

Comment:
It might be better to add the note outside of the processing model. (so right after the `</ol>`)